docs/api/context-bridge.md

```diff
@@ -0,0 +1,129 @@
+# contextBridge
+
+> Create a safe, bi-directional, synchronous bridge across isolated contexts
```

Comment:
Will this only ever be preload -> "main world"? This line (and the name contextBridge come to think of it) suggest that you can expose to arbitrary contexts, which, as far as I can tell, is not true.

Reply:
preload -> main world
and main world -> preload (bi-directional)

The code in theory could also bridge any worlds but currently it's only intended to support a bridge between these two.